.travis.yml

```diff
@@ -27,13 +27,16 @@ before_install:
 
     # enable gui, see https://docs.travis-ci.com/user/gui-and-headless-browsers
     - if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+            sudo apt update;
+            sudo apt -y install git;
             export DISPLAY=:99.0;
             sh -e /etc/init.d/xvfb start;
             sleep 3;
       fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew list python &>/dev/null || brew install python; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew list python3 &>/dev/null || brew install python3; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install pyenv-virtualenv; fi
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install git; fi
```

Comment:
It looks like this line is not needed for osx. The travis error says `Error: git 2.24.1 is already installed`

Reply:
I removed it, but I didn't want to assume that git is already installed. I didn't expect the pipeline to fail if `git` was already present to be honest.